### PR TITLE
Update getting started tutorial

### PR DESCRIPTION
**Pull request by dixia**
* correct out-of-date description

### CODE REVIEW

**Comment by dixia** [comment]
@bobgt when you got a moment mind to test if hello world tutorial code still works with this change?

File: docs/02_getting-started/03_smart-contract-development/01_hello-world.md [hunk 1]
```diff
@@ -42,7 +42,6 @@ using namespace eosio;
 
 class [[eosio::contract]] hello : public contract {
   public:
-  	 using contract::contract;
```
Comment (iamveritas) [inline docs/02_getting-started/03_smart-contract-development/01_hello-world.md:45]: this is redundant with what line? I've tried to build the eosio.cdt hello example without this line and I couldn't, it failed compilation.

Comment (dixia) [reply docs/02_getting-started/03_smart-contract-development/01_hello-world.md:45]: hi @iamveritas  this is a draft PL. Not ready for review yet. Also, see the [comment](https://github.com/EOSIO/welcome/pull/201#issuecomment-619760974) I made earlier

Comment (iamveritas) [reply docs/02_getting-started/03_smart-contract-development/01_hello-world.md:45]: I got email saying `@dixia requested your review on: #201 [draft PR] updated out-of-date description.`
that's why I looked at it. and I checked the files changed yes.

Comment (dixia) [reply docs/02_getting-started/03_smart-contract-development/01_hello-world.md:45]: no worry. 

could you try to use the code in `01_hello-world.md` to test? The change is based on the code from that file

also, which version of CDT are you using?

**Pull request merged**